tests/integration/replication-4.tcl

```diff
@@ -1,4 +1,4 @@
-start_server {tags {"repl"}} {
+start_server {tags {"repl network"}} {
```

Comment:
any reason you took just 2 of the replication tests and not all of them?
i guess to smoke out io thread bugs it's ok. but this tag can maybe used for denytags some day, only only as include tags.

Reply:
Yes more elaborate tagging is needed, this is just the first step making white listing possible.